georg-ui/src/components/Result.vue

```diff
@@ -9,7 +9,7 @@
     <template v-if="!isNewMarker">
       <v-list-item-content @click.prevent="onclick()">
         <v-list-item-title
-          v-bind:class="{ 'blue--text text--darken-2': isActive }"
+          v-bind:class="{ 'blue--text text--darken-2': isActive || isHovered }"
           >{{ name }}</v-list-item-title
         >
         <v-list-item-subtitle id="resultContent" class="text--primary">
```

Comment:
Not new, but I think it would be good to consider to fix it now: the id "resultContent" is repeated for several list items. And id:s should be unique in html. Otherwise ok to merge.